backend/src/test/groovy/pt/ulisboa/tecnico/socialsoftware/tutor/user/domain/CreateUserTest.groovy

```diff
@@ -239,11 +274,11 @@ class CreateUserTest extends SpockTest {
         user.getQuizAnswers().size() == previousNumberQuizAnswers + 1
     }
 
-    def "remove from course executions" (){
+    def "remove inactive user from course executions" (){
```

Comment:
It is necessary to check that the user does not exist
It is necessary to check that the authUser is also removed